client/src/components/PosterStack/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { useSprings } from 'react-spring'
+import { useDrag } from 'react-use-gesture'
+
+import Poster from './Poster'
+import { StackContainer } from './styled'
+import coco from '../../images/movie-posters/coco.jpg'
```

Comment:
since `coco` is an image path, I would recommend adding the word `path` to the end